lib/parse.js

```diff
@@ -120,45 +143,59 @@ function parsePlistXML (node) {
     // TODO: what should we do with text types? (CDATA sections)
 
   } else if (node.nodeName === 'key') {
-    if(isEmptyNode(node)) return null;
     res = '';
-    if(node.childNodes[0]) {
+    if(!isEmptyNode(node)) {
       res = node.childNodes[0].nodeValue;
     }
     return res;
   } else if (node.nodeName === 'string') {
     res = '';
-    if(isEmptyNode(node)) return null;
-    for (d=0; d < node.childNodes.length; d++) {
-      res += node.childNodes[d].nodeValue;
+    if(!isEmptyNode(node)) {
+      for (i=0; i < node.childNodes.length; i++) {
+        var type = node.childNodes[i].nodeType;
+        if (type === 3 || type === 4) {
```

Comment:
rather than using these magic numbers `3` and `4` can you define them as node type constants?